kvstores.go

```diff
@@ -51,7 +51,12 @@ func stringSlice(values []interface{}) []string {
 
 	for _, v := range values {
 		if v != nil {
-			converted = append(converted, conv.String(v))
+			val, err := conv.String(v)
+			if err != nil {
+				continue
```

Comment:
Better to return an error here. Skipping errors leads to debug hell. As go-conv makes a great job, if an error occurs that means Internet is probably broken and the end of the world is coming. Safer to catch this error.